index.ts

```diff
@@ -131,10 +150,9 @@ export interface Options {
 	transform?(path: string): string;
 }
 
-interface TsConfig {
-	compilerOptions: CompilerOptions;
+type TsConfig = {
 	outDir: string;
-}
+} & Omit<ts.CompilerOptions, 'outDir'>;
```

Comment:
Shouldn't it be like

```suggestion
} & { compilerOptions: Omit<ts.CompilerOptions, 'outDir'> };
```

Anyway, I thought having `outDir` next to `compilerOptions` in the tsconfig was wrong in the first place?